### DISCUSSION TITLE
使用Nextcloud WebDAV模式上传提示认证错误

### DISCUSSION THREAD

**Discussion by Cloudac7** [messense/aliyundrive-webdav#907] [Q&A] [answered]
### 问题描述

根据README中说明，在rclone中使用Nextcloud配置Webdav服务，在同步过程中产生如下报错：

```
2023/08/31 13:25:49 Failed to create file system for "aliyun://uploads/": the remote url looks incorrect. Note that nextcloud chunked uploads require you to use the /dav/files/USER endpoint instead of /webdav. Please check 'rclone config show remotename' to verify that the url field ends in /dav/files/USERNAME
```

若换用Owncloud或Other模式，则可以正确认证并同步，即已排除密码、URL等问题。因而可能是Nextcloud模式问题。
Rclone版本采用最新版本的docker镜像，经确认其他方案可以正确实现同步。

### 重现步骤

1. 配置Rclone: 
    ```
    docker run --rm -it --volume ~/rclone:/config/rclone --user $(id -u):$(id -g) rclone/rclone config
    ```
2. 按照提示信息配置Webdav，输入模式（选取Nextcloud）、账号、密码等信息
3. 采用Rclone进行同步：`rclone --no-update-modtime copy <file> aliyun://uploads/`

### 版本

2.3.2

### 运行平台

Docker

### 日志

```
2023/08/31 13:25:49 Failed to create file system for "aliyun://uploads/": the remote url looks incorrect. Note that nextcloud chunked uploads require you to use the /dav/files/USER endpoint instead of /webdav. Please check 'rclone config show remotename' to verify that the url field ends in /dav/files/USERNAME
```

**Comment by ghost** [comment]
同样的问题

**Comment by xinyangli** [comment, marked as answer]
在rclone选项中关闭[Nextcloud chunked upload](https://rclone.org/webdav/#webdav-nextcloud-chunk-size)可以绕开[对url的检查](https://github.com/rclone/rclone/blob/55c12c9a2d6f432ee5b3664dbc7e780aefec8bbe/backend/webdav/chunking.go#L53-L63)。

命令行选项：
```bash
rclone ls --webdav-nextcloud-chunk-size 0 <remotename>
```

也可以在配置文件中将webdav-nextcloud-chunksize设为0。
